src/oslogin_utils.cc

```diff
@@ -192,11 +192,28 @@ bool NssCache::GetNextGroup(BufferManager* buf, struct group* result, int* errno
   return ParseJsonToGroup(cached_passwd, result, buf, errnop);
 }
 
+// ParseJsonRoot must be declared early here, before it is used.
```

Comment:
`must be declared early here, before it is used` is basically describing the language behavior. Please describe why you need this `ParseJsonRoot()` instead of calling `json_tokener_parse()` directly in this function's callers, I can infer the intention based on the PR description but it won't be easily there for future readers of the code.

Reply:
Good thinking; I wrote a better description that explains "why" it's declared away from its friends.